### PR TITLE
Fixed #25351 -- Added example for database test settings to docs.

### PR DESCRIPTION
**Pull request by cacoze**

### CODE REVIEW

File: docs/ref/settings.txt [hunk 1]
```diff
@@ -665,8 +665,21 @@ TEST
...
 
 A dictionary of settings for test databases; for more details about the
-creation and use of test databases, see :ref:`the-test-database`. The
-following entries are available:
+creation and use of test databases, see :ref:`the-test-database`. An example of
+a simple settings file containing info about the test database configure is the following::
+
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql_psycopg2',
```
Comment (charettes) [inline docs/ref/settings.txt]: Use `'django.db.backends.postgresql'`.

**Comment by cacoze** [comment]
Thank you @charettes

**Comment by MoritzS** [comment]
Try to break the lines so that the lines don't exceed 79 characters.

File: docs/ref/settings.txt [hunk 2]
```diff
@@ -665,8 +665,22 @@ TEST
 Default: ``{}``
 
 A dictionary of settings for test databases; for more details about the
-creation and use of test databases, see :ref:`the-test-database`. The
-following entries are available:
+creation and use of test databases, see :ref:`the-test-database`.
+An example of a simple settings file containing info about the test database
```
Comment (timgraham) [inline docs/ref/settings.txt]: If this is meant to be a separate paragraph, it needs a newline above. If not, then you can flow "An example of" on the previous line.

**Comment by timgraham** [comment]
Please squash your commits when updating, thanks!

**Comment by cacoze** [comment]
I inserted the newline between the paragraphs. Thanks for the feedback @timgraham @MoritzS

**Comment by timgraham** [comment]
merged in cf99bae53af2ed8f73622bc8a8ed0331c272fda3, thanks!

**Pull request closed without merging**